host/platform/Windows/capture/DXGI/src/d3d12.c

```diff
@@ -346,7 +347,7 @@ static bool d3d12_copyFrame(Texture * parent, ID3D11Texture2D * src)
   HRESULT status;
 
   if (this->copySleep > 0)
-    Sleep(this->copySleep);
+    usleep((uint64_t)(this->copySleep * 1000));
```

Comment:
We have a custom `nsleep` handler that uses `NtDelayExecution` and is supposed to be more accurate.